src/components/MessagesList/MessagesGroup/AuthorAvatar.vue

```diff
@@ -0,0 +1,115 @@
+<!--
+  - @copyright Copyright (c) 2019 Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @author Marco Ambrosini <marcoambrosini@pm.me>
```

Comment:
Joas!

Reply:
I mean here you should put your name @nickvergessen

Reply:
done